### PR TITLE
Update README.md to RIT DEMO COFFEE-PYTHON formula

### LINKED ISSUE #6: Update RIT DEMO COFFEE-PYTHON formula README
### What would you like ?

- I would the **RIT DEMO COFFEE-PYTHON** formula to have a complete README file explaining what the formula does, with an example of its execution (GIF).

- Formula : https://github.com/ZupIT/ritchie-formulas-demo/tree/master/demo/coffee-python

### Why is this needed ?

- With the Ritchie Portal (ongoing) those READMEs will be used to share formulas information with the community, and the better they are, the easiest it will be for the community to use them.

### Examples

- https://github.com/ZupIT/ritchie-formulas-demo/tree/master/demo/hello-world
- https://github.com/ZupIT/ritchie-formulas/tree/master/github/publish/repo

### PR DESCRIPTION
**Pull request by Harirai**
## What I did
- Added description and demo to README.md of COFFEE-PYTHON 
Fixed #6 .
## How to verify it
- By checking the changes on my fork.
## What kind of change does this PR make

- [ ] Major
- [X] Minor
- [ ] Patch

## Description for the changelog
- Updated RIT DEMO COFFEE-PYTHON formula README

### CODE REVIEW

**Review by GuillaumeFalourd — APPROVED** [review]

File: demo/coffee-python/README.md [hunk 1]
```diff
@@ -1,19 +1,28 @@
 # coffee-python
 
-## command
+## Premisses 
+- [Python installed.](https://www.python.org/downloads/)
```
Comment (GuillaumeFalourd) [inline demo/coffee-python/README.md:4]: This is only a premisse if the user wants to execute the formula locally, on Docker it's not needed as the language is installed on the container through the DockerFile configuration.

**Comment by GuillaumeFalourd** [comment]
Thank you for contributing once again with our product @Harirai 👍 

I left a comment, if you wish to update the file to be a bit more accurate, and I already approved the PR 🚀

**Comment by Harirai** [comment]
Can anyone please guide me on "How to read the codebase".
For example, Like what happens and which code runs when we run `rit tutorial` or `rit build` etc.
Any resources/links or comments are much appreciated!

File: demo/coffee-python/README.md (hunk 1 above)
Comment (Harirai) [reply demo/coffee-python/README.md:4]: So should I write "(Only to run formula locally)" after "python installed.

**Comment by GuillaumeFalourd** [comment]
> Can anyone please guide me on "How to read the codebase".
> For example, Like what happens and which code runs when we run `rit tutorial` or `rit build` etc.
> Any resources/links or comments are much appreciated!

We don't have a `Developer Guide` yet with all details about how to check your implementations on the CLI repository @Harirai .

I explained it in another ISSUE on the CLI if you want to take a look: https://github.com/ZupIT/ritchie-cli/issues/575#issuecomment-703121025

Regarding formulas, you just need to run the `rit build formula` and select the formula you wish to test to update its binary and see your implementation live.

Feel free to ask anything if you have any question.

**Comment by Harirai** [comment]
Thank you for the quick response @GuillaumeFalourd!
I was asking that for getting familiar with any new project (Not specific to ritchie-cli).
Like what are the good practices to understand the code of any project?
Thank you for helping me. :smile:

**Comment by GuillaumeFalourd** [comment]
The current project (ritchie-formula-demo) depends on the ritchie-cli project to works, so you won't find many informations here beside what is on the README.

Generally, projects have a Developer Guide to help users to understand how it works. For ritchie-cli, we have a landing page: https://ritchiecli.io/ and a gitbook documentation: https://docs.ritchiecli.io/ that can help the user getting started with the product.

Practise is another way. For example, you can understand how to create automations with Ritchie by implementing the challenges our team created: https://docs.ritchiecli.io/challenges/explanation

**Comment by Harirai** [comment]
Thank you! @GuillaumeFalourd

**Pull request merged**